python-example-logs-api-extension/README.md

```diff
@@ -8,34 +9,41 @@ In this example, we start by developing a simple extension and then add the abil
 When the Lambda service sets up the execution environment, it runs the extension (logs_api_http_extension.py). This extension first registers as an extension and then subscribes to the Logs API to receive the logs via HTTP protocol. It starts an HTTP listener which receives the logs and processes them.
 
 ## Layer Setup Process
+
 The extensions .zip file should contain a root directory called `extensions/`, where the extension executables are located and another root directory called `python-example-extension/`, where the core logic of the extension  and its dependencies are located.
 
 Creating zip package for the extension:
+
 ```bash
-$ cd python-example-logs-api-extension
-$ chmod +x extensions/logs_api_http_extension.py
-$ zip -r extension.zip .
+cd python-example-logs-api-extension
+chmod +x extensions/logs_api_http_extension.py
+zip -r extension.zip .
```

Comment:
This line should be updated to only include the necessary files and directories, especially now that there are new files that are not needed for the extensions zip.